lib/ExpensiMark.js

```diff
@@ -628,4 +612,25 @@ export default class ExpensiMark {
         }
         return replacedText;
     }
+
+    /**
+     * Handle whitespace for the content of blockquote if the text matches the regex or else just return the original text
+     *
+     * @param {RegExp} regex
+     * @param {String} textToCheck
+     * @param {Function} replacement
+     *
+     * @returns {String}
+     */
+    formatWhitespaceForQuote(regex, textToCheck, replacement) {
+        if (textToCheck.match(regex)) {
+            // Remove '&gt;' and trim each line of quote content
+            let textToFormat = textToCheck.trim().split('\n').map(row => row.substr(4).trim()).join('\n');
```

Comment:
I don't see the point of trimming `textToCheck`. Can you please explain what purpose this serves?

Reply:
Because we concat `textToCheck` through
```js
textToFormat += `${textSplitted[i]}\n`;
```
So `textToCheck` ends with `\n` and to void the empty string item return by `split('\n')`, we need to trim it before split it by `\n`

Reply:
But we already trim the output in the next line

Reply:
Let's say we have textToCheck = '&gt; a\n', then textToCheck.split('\n') will return ['&gt; a', ''] if we don't trim it before splitting it. Hmm, we'll map `row => row.substr(4).trim()` for empty string through I just tested and found method call `''.substr(4)` won't return error.

Reply:
I'll remove it if you 👍 it

Reply:
I don't see much value added keeping it or removing it, let's remove just for code clarity

Reply:
updated